src/main/scala/com/rbmhtechnology/calliope/serializer/kafka/PayloadFormatSerializer.scala

```diff
@@ -48,8 +48,12 @@ class PayloadFormatSerializer[A <: AnyRef] private(serializer: PayloadSerializer
 }
 
 object PayloadFormatDeserializer {
-  def apply[A](payloadMapper: AnyRef => A)(implicit system: ActorSystem): PayloadFormatDeserializer[A] =
-    new PayloadFormatDeserializer[A](DelegatingStringManifestPayloadSerializer(system), payloadMapper)
+  def apply[A](payloadMapper: AnyRef => A, withTombstoneDeserializer: Boolean = false)(implicit system: ActorSystem): PayloadFormatDeserializer[A] = {
```

Comment:
additionally we should add a factory-method that can be supplied with a `PartialFunction` as `payload-mapper` and returns a `PayloadFormatDeserializer[Option[A]]`